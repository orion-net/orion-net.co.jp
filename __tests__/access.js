import React from "react"
import renderer from "react-test-renderer"

import Main from "../src/pages/access"

jest.mock("gatsby-plugin-image", () => {
  const StaticImageMock = () => <img>Mocked StaticImage component</img>

  return {
    StaticImage: StaticImageMock,
  }
})

jest.mock("../src/components/AccessMap", () => ({
  AccessMap: () => "Mocked map",
}))

describe("access", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Main />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
