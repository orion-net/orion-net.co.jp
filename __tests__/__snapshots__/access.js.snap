// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`access renders correctly 1`] = `
<div
  className="site"
  id="page"
>
  <header
    className="site-header sp-part-top sp-header2"
    id="masthead"
    role="banner"
  >
    <div
      className="sp-part-top sp-header-inner"
      id="masthead-inner"
    >
      <div
        className="sp-part-top sp-site-branding2"
        id="sp-site-branding2-1"
      >
        <h1
          className="site-title sp-part-top sp-site-title"
          id=""
        >
          <a
            href="index.html"
          >
            ビルダーサロン
          </a>
        </h1>
        <h2
          className="site-description sp-part-top sp-catchphrase"
          id=""
        >
          オリオンは川崎市高津区久末の美容院・ヘアーサロンです。
        </h2>
        <div
          className="extra sp-part-top sp-site-branding-extra"
          id="sp-site-branding-extra-1"
          style={
            Object {
              "minHeight": "20px",
            }
          }
        />
      </div>
    </div>
  </header>
  <div
    className="site-main sp-part-top sp-main"
    id="main"
  >
    <div
      className="sp-part-top sp-content-header"
      id="contenthead"
    >
      <a
        style={
          Object {
            "display": "block",
          }
        }
      >
        <img
          className="sp-part-top sp-image"
          id="sp-image-1"
          src="/img/top3.gif"
        />
      </a>
      <nav
        className="navigation-main button-menu sp-part-top sp-site-navigation horizontal"
        id="sp-site-navigation-1"
        role="navigation"
      >
        <h1
          className="menu-toggle"
        >
          メニュー
        </h1>
        <div
          className="screen-reader-text skip-link"
        >
          <a
            href="#content"
            title="コンテンツへスキップ"
          >
            コンテンツへスキップ
          </a>
        </div>
        <ul
          className="toggled-on"
          id="menu-mainnav"
        >
          <li
            className="menu-item "
          >
            <a
              aria-current="page"
              className=""
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              style={Object {}}
            >
              トップページ
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/concept"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              サロン紹介
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/menu"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              メニュー
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/custom"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              商品紹介
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/custom2"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              おすすめ
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/custom1"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              ご予約
            </a>
          </li>
          <li
            className="menu-item current_page_item"
          >
            <a
              href="/access"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              アクセス
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/concept1"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              成人式・七五三
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/contact"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              お問い合わせ
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/privacy"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              プライバシーポリシー
            </a>
          </li>
          <li
            className="menu-item "
          >
            <a
              href="/custom3"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              成人式・スナップ集
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="sp-part-top sp-bread-crumb"
        id="breadcrumb-list"
      >
        <div>
          <a
            aria-current="page"
            className=""
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
            style={Object {}}
          >
            トップ
          </a>
        </div>
        <div>
          ›
        </div>
        <div>
          アクセス
        </div>
      </div>
    </div>
    <div
      id="main-inner"
    >
      <div
        className="content-area"
        id="primary"
      >
        <div
          className="site-content sp-part-top sp-content page-access"
          id="content"
          role="main"
        >
          <header
            className="entry-header sp-part-top sp-page-title"
            id="sp-page-title-6"
          >
            <h1
              className="entry-title"
            >
              アクセス
            </h1>
          </header>
          <article>
            <div
              className="sp-part-top sp-block-container"
              id="page-content"
            >
              <h3
                className="paragraph"
              >
                周辺地図
              </h3>
              Mocked map
              <br />
              <p
                className="paragraph"
              >
                オリオン美容室
                <br />
                〒213-0026　神奈川県川崎市高津区久末２５３
                <br />
                TEL.044-766-1610　　ﾌﾘｰﾀﾞｲﾔﾙ .0800-800-1612
              </p>
              <p
                className="paragraph"
              >
                 
              </p>
              <hr />
              <h3
                className="paragraph"
              >
                アクセス
              </h3>
              <h4
                className="paragraph"
              >
                電車の場合
              </h4>
              <p
                className="paragraph"
              >
                JR南武線・東急東横線　武蔵小杉駅より　バス　道中坂下、鷺沼駅行き　バス停「久末」下車
                <br />
                田園都市線　鷺沼駅より　　　　　　　　バス　小杉駅前、新城駅前行き　バス停「久末」下車
              </p>
              <p
                className="paragraph"
              >
                JR南武線・武蔵新城駅より　　　　　　　バス　鷺沼駅行き　バス停「久末」下車
              </p>
              <p
                className="paragraph"
              >
                ※バス停「久末」から　徒歩1分　
              </p>
              <p
                className="paragraph"
              >
                　久末交差点を、左にガソリンスタンド、右に旅行会社の道を入りすぐ、右側にございます。
              </p>
              <h4
                className="paragraph"
              >
                自動車の場合
              </h4>
              <p
                className="paragraph"
              >
                ※駐車場２台
                <br />
                ※店の並び(1軒はさんだ隣）に駐車場スペースがございます。
              </p>
              <p
                className="paragraph"
              >
                　駐車場奥（竹やぶ側）にORIONの立て看板があります。そちらにお停め下さい。
              </p>
            </div>
          </article>
        </div>
      </div>
      <div
        className="sp-part-top sp-sidebar"
        id="sidebar1"
      >
        <div
          className="sp-part-top sp-button"
          id="sp-button-1"
        >
          <a
            href="https://ameblo.jp/teamorion/?frm_id=v.mypage-ameblo--myblog--blog"
          >
            ブログ
          </a>
        </div>
        <div
          className="sp-part-top sp-button"
          id="sp-button-2"
        >
          <a
            href="https://www.facebook.com/orionannex.orion"
          >
            Face Book
          </a>
        </div>
        <div
          className="sp-part-top sp-sns-button"
          id="sp-sns-button-1"
        >
          <ul>
            <li>
              <iframe
                allowtransparency="true"
                frameBorder="0"
                scrolling="no"
                src="https://www.facebook.com/plugins/like.php?href=http%3A%2F%2Forion-net.co.jp%2Fprivacy.html&layout=button_count&width=120&height=21&send=true&show_faces=true&share=false&action=like&colorscheme=light&font"
                style={
                  Object {
                    "border": "none",
                    "height": "21px",
                    "overflow": "hidden",
                    "width": "120px",
                  }
                }
              />
            </li>
            <li>
              <a
                className="twitter-share-button"
                data-count="horizontal"
                data-lang="ja"
                href="https://twitter.com/share"
              >
                Tweet
              </a>
              <script
                charSet="utf-8"
                src="https://platform.twitter.com/widgets.js"
                type="text/javascript"
              />
            </li>
            <li>
              <a
                href="https://line.me/R/msg/text/?%0D%0Ahttp%3A%2F%2Forion-net.co.jp%2Fprivacy.html"
              >
                <img
                  alt=""
                  height="20"
                  src="/img/linebutton_horizontal.png"
                  style={
                    Object {
                      "border": "none",
                    }
                  }
                  width="82"
                />
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <footer
    className="site-footer sp-part-top sp-footer2"
    id="colophon"
    role="contentinfo"
  >
    <div
      className="sp-part-top sp-footer-inner"
      id="colophon-inner"
    >
      <nav
        className="navigation-main sp-part-top sp-site-navigation minimal"
        id="sp-site-navigation-2"
        role="navigation"
      >
        <h1
          className="menu-toggle"
        >
          メニュー
        </h1>
        <div
          className="screen-reader-text skip-link"
        >
          <a
            href="#content"
            title="コンテンツへスキップ"
          >
            コンテンツへスキップ
          </a>
        </div>
        <ul
          id="menu-mainnav"
        >
          <li
            className="menu-item"
          >
            <a
              href="/privacy"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              プライバシーポリシー
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="sp-part-top sp-block-container"
        id="sp-block-container-1"
      >
        <p
          className="copyright paragraph"
        >
          Copyright © オリオン美容室, All rights reserved.
        </p>
      </div>
    </div>
  </footer>
</div>
`;
